.github/workflows/codegen-ci.yml

```diff
@@ -120,7 +98,7 @@ jobs:
           path: results/sdk-codegen
 
   publish-test-results:
-    name: Publish Test Results
+    name: Codegen Test Results
```

Comment:
Purposely named these `publish-test-results` jobs distinctly across workflow files so that that we can duplicate them as dummy-success jobs within the same file to satisfy the silly github shortfall regarding required checks (see code comment below and PR description for more detail)